components/Icon/Icon.jsx

```diff
@@ -39,6 +39,7 @@ import Contrast from '@mui/icons-material/Contrast';
 import CreditCard from '@mui/icons-material/CreditCard';
 import DateRange from '@mui/icons-material/DateRange';
 import Delete from '@mui/icons-material/Delete';
+import DescriptionOutlinedIcon from '@mui/icons-material/DescriptionOutlined';
```

Comment:
```suggestion
import DescriptionOutlined from '@mui/icons-material/DescriptionOutlined';
```

Reply:
I believe that the "Icon" prefix is ​​not necessary, it is somewhat redundant. How to improve, I suggest removing it